config/settings.py

```
@@ -12,11 +12,11 @@
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
 
-# SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = "django-insecure-rx29wzwm)_#)bv+=m(81-^aw$qm=xap)29ie2wn$)geae-q91)"
+SECRET_KEY = os.environ.get("DJANGO_SECRET_KEY", None)
+if SECRET_KEY is None:
+    raise ValueError("DJANGO_SECRET_KEY environment variable is not set")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)